litellm/llms/vertex_ai/gemini/vertex_and_google_ai_studio_gemini.py

```diff
@@ -167,6 +167,8 @@ def get_supported_openai_params(self):
             "n",
             "stop",
             "extra_headers",
+            "frequency_penalty",
```

Comment:
@mbukeRepo the vertexgemini config (below) is the one which is used for vertex ai gemini models. 

That already shows support for frequency + presence penalty - https://github.com/BerriAI/litellm/blob/bd4ab1449e78a2d4bc209ef7c781c018add9f8e1/litellm/llms/vertex_ai/gemini/vertex_and_google_ai_studio_gemini.py#L343

how do you see the error?